sdk/core/loader/boot.S

```diff
@@ -89,8 +89,19 @@ start:
 	cspecialrw		ca3, mscratchc, ca3
 	// ca4 still has the RW memory root; nothing to change
 	// The return value is SchedEntryInfo.
-	cincoffset		csp, csp, -16 - CONFIG_THREADS_NUM * BOOT_THREADINFO_SZ
-	csetbounds		ca0, csp, 16 + CONFIG_THREADS_NUM * BOOT_THREADINFO_SZ
+
+	// Calculate space for the structure passed from the loader to the
+	// scheduler.  This is a `class ThreadInfo` from `loader/types.h`.
```

Comment:
Ooops, I shouldn't have committed this proposed change.  This is actually a 	`SchedulerEntryInfo` from `boot.cc`.

```suggestion
```